scripts/deploy/github/build-images.sh

```diff
@@ -20,14 +20,31 @@ set -e
 
 REGISTRY="${REGISTRY:-kind-registry:5000}"
 TAG="${TAG:-latest}"
+EXIT_CODE=0
 
 docker system prune -a -f
 
-docker build -q -t "${REGISTRY}/apiserver:${TAG}" -f backend/Dockerfile . && docker push "${REGISTRY}/apiserver:${TAG}" &
-docker build -q -t "${REGISTRY}/persistenceagent:${TAG}" -f backend/Dockerfile.persistenceagent . && docker push "${REGISTRY}/persistenceagent:${TAG}" &
-docker build -q -t "${REGISTRY}/scheduledworkflow:${TAG}" -f backend/Dockerfile.scheduledworkflow . && docker push "${REGISTRY}/scheduledworkflow:${TAG}" &
+dacker build -q -t "${REGISTRY}/apiserver:${TAG}" -f backend/Dockerfile . && docker push "${REGISTRY}/apiserver:${TAG}" || EXIT_CODE=$?
```

Comment:
s/dacker/docker/

Reply:
i did that in order for the action to fail so i could see the issue, i changed it back and broke the DockerFile instead